src/Tools/dotnet-dsrouter/Program.cs

```diff
@@ -61,6 +63,18 @@ internal class Program
                 IpcServerAddressOption(), TcpClientAddressOption(), RuntimeTimeoutOption(), VerboseOption(), ForwardPortOption()
             };
 
+        private static Command IpcServerWebSocketServerRouterCommand() =>
+        new Command(
+            name: "server-websocket",
```

Comment:
would support for client-websocket mode make sense as well? Since you already have the websocket server supporting client-websocket is mainly building the router out of the components. dotnet-monitor's support for startup tracing is mainly utilize the reverse diagnostic server and logic around the fact that the process will connect to dotnet-monitor.

Reply:
makes sense, I can try it.